src/components/v2/appDetails/k8Resource/nodeType/NodeTree.component.tsx

```diff
@@ -159,10 +163,13 @@ export function generateSelectedNodes(
         if (parents.length === 2 || (parents.length === 1 && _nodeLowerCase !== NodeType.Pod.toLowerCase())) {
             // remove if leaf node selected previously if any
             let _childNodes = _treeNodes.flatMap((_tn) => _tn.childNodes ?? []);
-            let leafNode = _childNodes.find(
-                (_cn) =>
-                    clickedNodes.has(_cn.name.toLowerCase()) && _cn.name.toLowerCase() !== NodeType.Pod.toLowerCase(),
-            );
+            let leafNode = !isDevtronApp
```

Comment:
Can we write it as let leafNode = _childNodes.find(
                      (_cn) =>
                          clickedNodes.has(_cn.name.toLowerCase()) &&
                          (isDevtronApp || _cn.name.toLowerCase() !== NodeType.Pod.toLowerCase()),
                  )

Reply:
Done